modules/sbo-deploying-a-postgresql-database-operator-power-z.adoc

```diff
@@ -0,0 +1,74 @@
+[id="sbo-deploying-a-postgresql-operator-instance-power-z_{context}"]
+= Deploying a PostgreSQL Operator
+
+To deploy the `dev4devs PostgreSQL Operator` in the `my-postgresql` namespace run the following command in shell:
+
+[source,terminal]
+----
+$ oc apply -f - << EOD
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-postgresql
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: postgres-operator-group
+  namespace: my-postgresql
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: ibm-multiarch-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: quay.io/ibm/operator-registry-<architecture> <1>
+  imagePullPolicy: IfNotPresent
+  displayName: ibm-multiarch-catalog
+  updateStrategy:
+    registryPoll:
+      interval: 30m
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: postgresql-operator-dev4devs-com
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: postgresql-operator-dev4devs-com
+  source: ibm-multiarch-catalog
+  sourceNamespace: openshift-marketplace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: database-view
+  labels:
+    servicebinding.io/controller: "true"
+rules:
+  - apiGroups:
+      - postgresql.dev4devs.com
+    resources:
+      - databases
+    verbs:
+      - get
+      - list
+EOD
+----
+<1> The Operator image. 
+* For IBM Power Systems: `quay.io/ibm/operator-registry-ppc64le:release-4.9` 
+* For IBM Z and LinuxONE: `quay.io/ibm/operator-registry-s390x:release-4.8`
+
+.Example output
+[source,terminal]
+----
+$ oc get subs -n openshift-operators
+NAME                               PACKAGE                            SOURCE                  CHANNEL
```

Comment:
Just a minor change. To be consistent with the other cut and paste examples if you could make the command separate from the output so the copy works correctly.